src/cdk/text-field/autosize.ts

```diff
@@ -58,6 +63,10 @@ export class CdkTextareaAutosize implements AfterViewInit, DoCheck, OnDestroy {
     this._setMaxHeight();
   }
 
+  /** Emit event when textarea height changes */
+  @Output('cdkAutosizeSizeChanged')
+  sizeChanged: EventEmitter<number> = new EventEmitter<number>();
```

Comment:
We generally prefer emitting event objects instead of primitives since it's easier to add more information later. In this case, you could emit both the old and the new values.